docs/source/osds/scenarios.rst

```diff
@@ -1,21 +1,25 @@
-OSD Scenarios
-=============
+OSD Scenario
+============
 
-There are a few *scenarios* that are supported and the differences are mainly
-based on the Ceph tooling required to provision OSDs, but can also affect how
-devices are being configured to create an OSD.
+As of stable-3.2, the following scenarios are not supported anymore since they are associated to ``ceph-disk``:
 
+* `collocated`
+* `non-collocated`
+
+``ceph-disk`` was deprecated during the ceph-ansible 3.2 cycle and has been removed entirely from Ceph itself in the Nautilus version.
 Supported values for the required ``osd_scenario`` variable are:
 
-* :ref:`collocated <osd_scenario_collocated>`
-* :ref:`non-collocated <osd_scenario_non_collocated>`
+At present (starting from stable-3.2), there is only one scenario, which defaults to ``lvm``, see:
+
 * :ref:`lvm <osd_scenario_lvm>`
 
-Since the Ceph mimic release, it is preferred to use the :ref:`lvm scenario
+So there is no need to configure ``osd_scenario`` anymore, it defaults to ``lvm``.
```

Comment:
does it make sense to keep osd_scenario variable if we only have one possible value ?